src/timetrackings/types.ts

```diff
@@ -81,6 +84,15 @@ export interface TimeTracking {
   geo_long?: number;
   geo_accuracy?: number;
   updated?: string;
+  status_invoicing?: number;
+  start_time__timezone_hours_to_add?: string;
+  end_time__timezone_hours_to_add?: string;
+  invoice_id?: string;
```

Comment:
Type is number, and possibly never undefined.

Reply:
Many members which are never undefined, i.e. timezone are also marked partial... I'm not sure why. How do you suggest I rewrite this type, invoice_id: number | null ?

Reply:
Initially we had plenty undefined, because we have used one type across everything.

Correct way would be to split it:
- entity
- read (possible read params)
- update (possible update params)
- create (possible create params)

But this will probably get addressed with automatic script for type generation, at some point I or someone else will address it.

 I suggest to rewrite type as unvoice_id: number, I have checked and nullable is not possible for invoice_id.

Reply:
SB-819, change made

Reply:
but you didn't change to  invoice_id: number but to  invoice_id?: number how come?